com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraph.cs

```diff
@@ -1535,6 +1537,33 @@ void RenderGizmos(RenderGraph renderGraph, HDCamera hdCamera, TextureHandle colo
             return executed;
         }
 
+        class ResetCameraSizeForAfterPostProcessPassData
+        {
+            public HDCamera hdCamera;
+            public ShaderVariablesGlobal shaderVariablesGlobal;
+        }
+
+        void ResetCameraSizeForAfterPostProcess(RenderGraph renderGraph, HDCamera hdCamera, CommandBuffer commandBuffer)
+        {
+            if (hdCamera.allowDynamicResolution && HDUtils.hdrpSettings.dynamicResolutionSettings.enabled)
```

Comment:
Better to go through the Dynamic resolution handler method: DynamicResolutionEnabled ( `DynamicResolutionHandler.instance.DynamicResolutionEnabled ` )

Reply:
Thanks, fixed :)